ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/components/BorrowingRequestPatronLoan/BorrowingRequestPatronLoan.js

```diff
@@ -0,0 +1,267 @@
+import { fromISO, toShortDate } from '@api/date';
+import { DatePicker } from '@components';
+import { getDisplayVal } from '@config/invenioConfig';
+import { MetadataTable } from '@pages/backoffice/components';
+import { LoanIcon } from '@pages/backoffice/components/icons';
+import { BackOfficeRoutes } from '@routes/urls';
+import _get from 'lodash/get';
+import { DateTime } from 'luxon';
+import { PropTypes } from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  Divider,
+  Form,
+  Grid,
+  Icon,
+  Label,
+  Message,
+  Modal,
+} from 'semantic-ui-react';
+
+const WrongStatus = () => (
+  <Message>
+    <Message.Content>
+      <Message.Header>Create new loan</Message.Header>
+      To create a new loan, the borrowing request must have the status{' '}
+      <strong>REQUESTED</strong>.
+    </Message.Content>
+  </Message>
+);
+
+const HasAlreadyLoan = () => (
+  <Message warning>
+    <Message.Content>
+      <Message.Header>Loan already created</Message.Header>
+      The borrowing request has already a loan. To create a new one, edit the
```

Comment:
We can make `edit` a link, to jump on loan edit directly.

Reply:
The link is just above, on the loan PID, see screenshot.